src/components/application_manager/src/rpc_handler_impl.cc

```diff
@@ -261,7 +259,8 @@ bool RPCHandlerImpl::ConvertMessageToSO(
       }
 
       if (!conversion_result ||
-          !mobile_so_factory().attachSchema(output, true, msg_version) ||
```

Comment:
@IGapchuk is it style change?

Reply:
@LuxoftAKutsan No, it is not a style change. Before there changes, rps_handler had the hmi_so_factory and mobile_so_factory as members, and he had private getters for there instances. After there changes, rpc_handler has only v4_protocol_so_factory as member. hmi_so_factory and mobile_so_factory exists in application manager only and rpc_handler has access for them by application manager interface.